### PR TITLE
update to use 2 spaces for indentation

### PR DESCRIPTION
**Pull request by nilslp**
Narrative converted to use 2 spaces for indentation.

### CODE REVIEW

File: properties.schema [hunk 1]
```diff
@@ -43,7 +43,7 @@
     "_hasNavigationInTextArea": {
       "type": "boolean",
       "required": true,
-      "default": true,
+      "default": false,
```
Comment (oliverfoster) [inline properties.schema]: this is the only anomaly, it makes sense to me, where did it come from?

File: bower.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "adapt-contrib-narrative",
-  "version": "4.0.0",
+  "version": "5.0.0",
```
Comment (oliverfoster) [inline bower.json]: How come the version bump?

Comment (nilslp) [reply bower.json]: I branched off master (it had a version bump, that v5 branch doesnt have).

File: properties.schema (hunk 1 above)
Comment (oliverfoster) [reply properties.schema]: merge from master as discussed on another comment

**Pull request merged**